gradle/libs.versions.toml

```diff
@@ -24,6 +24,7 @@ guava             = "32.1.1-jre"
 protobuf          = "3.25.5"
 jakarta_annotation = "1.3.5"
 google_http_client = "1.44.1"
+google_auth       = "1.29.0"
```

Comment:
@reta , I've moved the version specification to this file, so that  plugins can use same version, please help to take a look, thanks!